### PR TITLE
Committers To Ignore Null Pointer

### PR DESCRIPTION
**Pull request by casz**
fixes null pointer created in #78

### CODE REVIEW

**Comment by PixelJonas** [comment]
This is getting embarassing ;) Srry for the missing `null` check 😇

**Comment by casz** [comment]
@PixelJonas blame the reviewer for not catching it 😥

**Comment by PixelJonas** [comment]
I guess we should work a bit on the testing part of the plugin to catch this kind of stuff. Unfortunatly I only have the Atlassian-SDK as a testing env for me.

Is there some kind of circle-ci/travis integration for the Atlassian-SDK where we could add some integration tests?

**Comment by casz** [comment]
@PixelJonas We are already using Travis to run integration test on a matrix,
https://github.com/Eernie/bitbucket-webhooks-plugin/blob/master/.travis.yml

See build results here:
https://travis-ci.org/Eernie/bitbucket-webhooks-plugin

Sadly the tests are not covering everything as you saw :) I had to edit the nullConfig unit test to reveal the removed null pointer check 👍

File: src/main/java/nl/topicus/bitbucket/api/PushEventService.java [hunk 1]
```diff
@@ -31,18 +31,20 @@ private boolean isIgnoredEvent(Ignorable event, WebHookConfiguration configurati
...
-                    configuration.getRepositoryId(),
-                    event.getUsername(),
-                    configuration.getTitle(),
-                    configuration.getURL(),
-                    configuration.getCommittersToIgnore());
-            return true;
+        if (configuration.getCommittersToIgnore() != null) {
+            List<String> ignoredCommitersList = Arrays.asList(configuration.getCommittersToIgnore().split("\\s?,\\s?"));
+            if (event.getUsername().isPresent()
+                    && ignoredCommitersList.size() > 0
```
Comment (abishek3876) [inline src/main/java/nl/topicus/bitbucket/api/PushEventService.java]: This is a redundant check. It also doesn't give noticeable performance gain AFAIK.
